src/wp-signup.php

```diff
@@ -197,19 +204,24 @@ function show_blog_form( $blogname = '', $blog_title = '', $errors = '' ) {
 	?>
 
 	<div id="privacy">
-		<p class="privacy-intro">
-			<?php _e( 'Privacy:' ); ?>
-			<?php _e( 'Allow search engines to index this site.' ); ?>
-			<br style="clear:both" />
-			<label class="checkbox" for="blog_public_on">
-				<input type="radio" id="blog_public_on" name="blog_public" value="1" <?php echo $blog_public_on_checked; ?> />
-				<strong><?php _e( 'Yes' ); ?></strong>
-			</label>
-			<label class="checkbox" for="blog_public_off">
-				<input type="radio" id="blog_public_off" name="blog_public" value="0" <?php echo $blog_public_off_checked; ?> />
-				<strong><?php _e( 'No' ); ?></strong>
-			</label>
-		</p>
+		<fieldset class="privacy-intro">
+			<legend>
+				<p>
```

Comment:
`legend` can only contain phrasing content and headings; a paragraph is flow content. Can we change this into a heading?

Reply:
Perhaps it could be an H3, though I used the paragraph for consistent styles and H3 would enlarge the legend text for many sites. Simply removing the P tag (and its related style) might be better.

Reply:
For now, the paragraphs are simply removed from `legend` elements.

If you meant using a heading for only the "Privacy:" text string, I tried switching the `span` to `h3` (still with the `label-heading` class). I'm not sure it is worth the change, but that would also include `inherit` for some CSS properties such as `font-family` and `letter-spacing`, perhaps even `color`.

Without `inherit`, H3 would make some style changes in about half the bundled themes ([see screenshots for comparison](https://user-images.githubusercontent.com/17100257/186254828-27a33885-2f18-479d-8ff9-808dc32f9412.png)):
- Different `font-family` and `line-height` in Twenty Thirteen
- Different `line-height` in Twenty Fourteen and Twenty Seventeen
- Different `font-family`, `letter-spacing` and `line-height` in Twenty Nineteen
- Different `letter-spacing` and `line-height` in Twenty Twenty
- Different `line-height` with Twenty Twenty-One, which can have a different `font-family` in Hebrew
- No difference for the other five "classic" bundled themes

Reply:
I left the element as a `span` so we would not need to decide which properties to inherit right now. If it's worth changing to an H3, it could be addressed on a separate ticket. And I made screenshots to compare the [heading style difference in Twenty Twenty-Two](https://user-images.githubusercontent.com/17100257/189962077-06f4ee35-7a30-43fa-8df5-525b2daf8422.png) of `font-family` and `line-height`.